recipe/build.sh

```diff
@@ -1,6 +1,12 @@
 set -e
 
-./configure --prefix=$PREFIX --enable-official-khronos-headers
+./bootstrap
+
+./configure \
+  --prefix=$PREFIX \
+  --enable-official-khronos-headers \
+  --enable-custom-vendordir=$PREFIX/etc/OpenCL/vendors
```

Comment:
Also is there a way to still include the system in the search path here? For instance, if I have NVIDIA's OpenCL installed, that is living in `/etc/OpenCL/vendors/`. Right now the system directory is overlooked. Admittedly we probably want the system path after the conda environment one.